src/ast.mli

```diff
@@ -0,0 +1,91 @@
+type program = None
+
+type bin_op =
+  Add    | Minus | Multiply | Divide
+  | Modulo | Equal | Neq      | Lt
+  | Leq    | Gt    | Geq      | And
+  | Or
+
+type u_op = Not | Neg
+
+type p_type = Int_t | Bool_t | Double_t | Char_t | Unit_t
+
+type cont_type = String_t | List_t | Set_t | Map_t | Tuple_t
+
+type actor_op =
+  | Actor_send
+  | Actor_broadcast
+  | Actor_receive
+
+type types =
```

Comment:
<img width="517" alt="screen shot 2016-10-29 at 2 47 11 pm" src="https://cloud.githubusercontent.com/assets/22507776/19832134/da05e6a6-9de9-11e6-98ba-45f8633b3088.png">
I think the code needs to be changed to this^